spec/features/admin_feature_spec.rb

```diff
@@ -108,6 +108,26 @@
       end
     end
 
+    context 'with local delivery units', :js do
+      before do
+        visit('/admin/local_delivery_units')
+      end
+
+      it 'can create one' do
+        expect {
+          click_link 'New Localdeliveryunit'
+          sleep 1
+          fill_in 'Code', with: 'TN01'
+          fill_in 'Name', with: 'Thing'
+          fill_in 'Email address', with: 'test@example.com'
+          select 'England'
+          click_button 'Create Local delivery unit'
+          sleep 1
+        }.to change(LocalDeliveryUnit, :count).by(1)
+        sleep 20
```

Comment:
Are any of these `sleep`s needed?

The `sleep 20` in particular seems like it might be a left-over from debug/dev?